src/controllers/__tests__/user.test.js

```diff
@@ -0,0 +1,97 @@
+const request = require('supertest');
+
+const server = require('../../app');
+
+const {
+  closeDatabase,
+  clearDatabase,
+  connectToMongo,
+} = require('../../db/connection');
+
+let userId;
+
+const newValidUser = {
+  username: 'new.user',
+  firstName: 'New',
+  lastName: 'User',
+  email: 'email@domain.com',
+  password: 'testPassword',
+  password2: 'testPassword',
+  address: 'new address',
+  acceptTos: 'on',
+};
+
+const newValidUser2 = {
+  username: 'new.user2',
+  firstName: 'New2',
+  lastName: 'User2',
+  email: 'email2@domain.com',
+  password: 'testPassword2',
+  password2: 'testPassword2',
+  address: 'new address2',
+  acceptTos: 'on',
```

Comment:
Is `acceptTos` property used in your authentication logic?

Reply:
It is implemented in POST /signup.